3ds Max/Max2Babylon/Exporter/BabylonExporter.cs

```diff
@@ -444,15 +438,15 @@ public void Export(ExportParameters exportParameters)
                 {
                     ExportSkin(skin, babylonScene);
                 }
-            }
+            }
+
+            // Animation group
+            RaiseMessage("Export animation groups");

```

Comment:
Not sure why we have two paths for exporting animation groups between babylon/gltf. My solution was to just have the gltf exporter use the babylon scene's already exported animation groups.